POEApi.Model/ProxyMapper.cs

```diff
@@ -200,6 +200,192 @@ internal class ProxyMapper
+            {"The Sephirot", DivinationCardType.TheSephirot},
+            {"The Sigil", DivinationCardType.TheSigil},
+            {"The Siren", DivinationCardType.TheSiren},
+            {"The Soul", DivinationCardType.TheSoul},
+            {"The Spark and the Flame", DivinationCardType.TheSparkAndTheFlame},
+            {"The Spoiled Prince", DivinationCardType.TheSpoiledPrince},
+            {"The Standoff", DivinationCardType.TheStandOff},
+            {"The Stormcaller", DivinationCardType.TheStormcaller},
+            {"The Summoner", DivinationCardType.TheSummoner},
+            {"The Sun", DivinationCardType.TheSun},
+            {"The Surgeon", DivinationCardType.TheSurgeon},
+            {"The Surveyor", DivinationCardType.TheSurveyor},
+            {"The Survivalist", DivinationCardType.TheSurvivalist},
+            {"The Thaumaturgist", DivinationCardType.TheThaumaturgist},
+            {"The Throne", DivinationCardType.TheThrone},
+            {"The Tower", DivinationCardType.TheTower},
+            {"The Traitor", DivinationCardType.TheTraitor},
+            {"The Trial", DivinationCardType.TheTrial},
+            {"The Twins", DivinationCardType.TheTwins},
+            {"The Tyrant", DivinationCardType.TheTyrant},
+            {"The Union", DivinationCardType.TheUnion},
+            {"The Valkyrie", DivinationCardType.TheValkyrie},
+            {"The Valley of Steel Boxes", DivinationCardType.TheValleyOfSteelBoxes},
+            {"The Vast", DivinationCardType.TheVast},
+            {"The Visionary", DivinationCardType.TheVisionary},
+            {"The Void", DivinationCardType.TheVoid},
+            {"The Warden", DivinationCardType.TheWarden},
+            {"The Warlord", DivinationCardType.TheWarlord},
+            {"The Watcher", DivinationCardType.TheWatcher},
+            {"The Web", DivinationCardType.TheWeb},
+            {"The Wind", DivinationCardType.TheWind},
+            {"The Wolf", DivinationCardType.TheWolf},
+            {"The Wolf's Shadow", DivinationCardType.TheWolfsShadow},
+            {"The Wolven King's Bite", DivinationCardType.TheWolvenKingsBite},
+            {"The Wolverine", DivinationCardType.TheWolverine},
+            {"The Wrath", DivinationCardType.TheWrath},
+            {"The Wretched", DivinationCardType.TheWretched},
+            {"Three Faces in the Dark", DivinationCardType.ThreeFacesInTheDark},
+            {"Thunderous Skies", DivinationCardType.ThunderousSkies},
+            {"Time-Lost Relic", DivinationCardType.TimeLostRelic},
+            {"Tranquillity", DivinationCardType.Tranquillity},
+            {"Treasure Hunter", DivinationCardType.TreasureHunter},
+            {"Turn the Other Cheek", DivinationCardType.TurnTheOtherCheek},
+            {"Vinia's Token", DivinationCardType.ViniasToken},
+            {"Volatile Power", DivinationCardType.VolatilePower},
+            {"Wealth and Power", DivinationCardType.WealthAndPower}
+        };
```

Comment:
Is there a way we can omit the need to maintain long lists of all possible divination cards? I never liked the way it was done.